tasks/getHistoricalData.ts

```diff
@@ -320,19 +326,35 @@ task("getHistoricalData", "Retrieves the historical rates")
 
         if (b >= aaveLendingPoolStartBlock) {
           try {
-            const r = await aavePool.getReserveNormalizedIncome(
-              aTokenUnderlyingAddresses[
-                asset as keyof typeof aTokenUnderlyingAddresses
-              ],
-              {
-                blockTag: b,
-              }
-            );
+            if (taskArgs.rate === "borrow") {
+              const r = await aavePool.getReserveNormalizedVariableDebt(
+                aTokenUnderlyingAddresses[
+                  asset as keyof typeof aTokenUnderlyingAddresses
+                ],
+                {
+                  blockTag: b,
+                }
+              );
 
-            blocks.push(b);
-            timestamps.push(block.timestamp);
-            rates.push(r);
-            fetch = FETCH_STATUS.SUCCESS;
+              blocks.push(b);
+              timestamps.push(block.timestamp);
+              rates.push(r);
+              fetch = FETCH_STATUS.SUCCESS;
+            } else {
```

Comment:
I think we go through here for any non-borrow value. Flag may be better but if using a param I'd probably default value to "lending"and have an eye branch that throws an error if neither expected value is received.